src/main/target/OMNIBUSF4/target.h

```diff
@@ -249,6 +249,8 @@
 #define RANGEFINDER_HCSR04_ECHO_PIN        PA8
 #define USE_RANGEFINDER_TF
 
+#define USE_DSHOT_DMAR
```

Comment:
Should this rather be defined for all F4 (potentially with exceptions for boards known to not work with it)?

Reply:
Yeah, I moved it to `common_fc_pre.h`.

Also prepared #4847 for collision reporting.